src/clang-c-frontend/clang_c_adjust_expr.cpp

```diff
@@ -522,7 +522,7 @@ void clang_c_adjust::adjust_type(typet &type)
 
     if(s == nullptr)
     {
-      log_error("type symbol `{}' not found", identifier);
+      log_error("type symbol `{}' not found in {}", identifier, __func__);
```

Comment:
Could you put the source location info in front of the actual message in order to clarify that it's not part of that message itself, e.g.
```c++
log_error("{}:{} type symbol `{}' not found", __func__, __LINE__, identifier);
```

Reply:
Done.